bundle/jsky.app.ot/src/main/scala/jsky/app/ot/gemini/ghost/GhostEditor.scala

```diff
@@ -249,7 +536,58 @@ final class GhostEditor extends ComponentEditor[ISPObsComponent, Ghost] {
         toc.setTargetEnvironment(env)
         oc.setDataObject(toc)
         asterismComboBox.enabled = true
+        initIFUs()
+      }
+    }
+
+    /**
+     * When the guide fiber state changes, we must change the asterism type to reflect this.
+     */
+    // Get the state from the checkbox.
+    def gfs(box: CheckBox): GuideFiberState =
+      //box.selected ? GuideFiberState.Enabled | GuideFiberState.Disabled
+      if (box.selected) GuideFiberState.Enabled
+      else GuideFiberState.Disabled
+
+    /**
+     * Change the guide fiber type of the IFU1s.
+     */
+    def changeIFU1GuideFiberState(state: GuideFiberState): Unit =
+      for {
+        oc  <- Option(getContextTargetObsComp)
+        toc <- Option(oc.getDataObject).collect { case t: TargetObsComp => t }
+      } {
+        println("*** changeIFU1GuideFiberState")
```

Comment:
Let's make a note to remove this if not remove it now (and a few other similar printlns).